### PR TITLE
10 play uploaded music

### LINKED ISSUE #10: 10-play-uploaded-music
As a user,
So that i can hear new music,
I would like to be able to play music that has been uploaded

### PR DESCRIPTION
**Pull request by MarcoBgn**
closes #10

### CODE REVIEW

File: back-end/spec/features/streams_a_track_spec.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
-feature 'Tracks the number of plays a song has' do
+ENV['RAILS_ENV']='development'
 
-#defering feature tests until after implemnetation of user story 10: play uploaded music.
+feature 'Tracks the number of plays a song has' do
 
   scenario 'Logged in user plays a song already streamed', js: true do
-    Stream.find_by(track_id: 1).destroy!
```
Comment (drjparry) [inline back-end/spec/features/streams_a_track_spec.rb]: Feature name could be a little bit clearer. Perhaps 'logged in user can play a song that's already streamed'

File: back-end/spec/features/play_uploaded_music_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+feature 'Play Uploaded Music' do
+	scenario 'User uploads and plays uploaded track', js: true do
+		visit 'http://localhost:9000/#/account'
+		upload_track
+		click_link('Songs')
+		sleep(10)
+		click_button('1')
+		expect(page).to have_css("#song-name", text: "default track")
+	end
+
```
Comment (drjparry) [inline back-end/spec/features/play_uploaded_music_spec.rb]: Indentation should come in here

File: front-end/app/scripts/services/tracksService.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+fairMuse.service('TracksService', function($resource) {
+			return $resource('http://localhost:3000/tracks');
```
Comment (yanyil) [inline front-end/app/scripts/services/tracksService.js]: indentation

File: front-end/test/spec/services/trackService.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+	module('ui.bootstrap');
+
+	inject(function(_$httpBackend_, _TracksService_ , $q){
+    deferred = $q.defer();
+		httpBackend = _$httpBackend_
+		TracksService = _TracksService_
+		httpBackend.expectGET('views/main.html').respond(200)
+		httpBackend.flush()
+	});
+});
```
Comment (drjparry) [inline front-end/test/spec/services/trackService.js]: Indentation looks to be only a single space here. Line 10 and 15...

File: front-end/test/spec/services/trackService.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+	});
+  
+	it('returns a JSON object', function(){
+		httpBackend.whenGET('http://localhost:3000/tracks').respond(data)
+		TracksService.query(function(){data});
+    deferred.promise.then(function(data) {
+      httpBackend.flush()
+    expect(httpBackend.flush).not.toThrow();
+  	});
+    })
```
Comment (drjparry) [inline front-end/test/spec/services/trackService.js]: indentation needs to be corrected here. 31 is further out than 30

**Comment by drjparry** [comment]
:shipit:

**Pull request merged**